modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,86 @@
+This means that any keyspace that your client application uses must exist on both Origin and Target with the same name. 
+Table names must also match. For more, see xref:feasibility-checklists.adoc#_schemakeyspace_compatibility[Schema/keyspace compatibility].
+====
+
+Your migration project occurs through a sequence of phases. 
+Here is an example of these phases with ZDM using ZDM proxy, which is a simple and lightweight proxy that handles all the real-time requests generated by your client applications.
+For more, see http:addlinkhere.adoc[ZDM proxy]
+
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+
+Before walking through illustrations of each phase, look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.  
+
+image::{imagesprefix}zdm-migration-before-starting.png[Diagram shows existing CQL-compatible environment before migration starts.]
+
+Before any migration begins, complete prerequisites, prepare your environment, and set up the recommended infrastructure.
+////
+
+== Migrations with and without zero downtime examples
+
+Data migrations with downtime are those where the data being migrated is critical for ongoing operations, and the migration process itself requires systems to be taken offline temporarily. Downtime may be required for:
+
+* Migration for large-scale systems: these systems may require downtime for completely smooth transition.
+* Database migration: large database migrations or database software upgrades may need downtime for data consistency and integrity.
+* Storage system migration: migrating from on-prem to cloud may need downtime to reconfigure systems.
+* Network infrastructure changes: these systems may require downtime for completely smooth transition.
+
+A migration with no downtime is seamless and ideal, but doesn't always happen. This happens when your business can tolerate a brief outage. 
+Here are scenarios for such migration:
+ 
+include::{imagesprefix}partial$migration-scenarios.adoc[]
+
+== Data migration tools
+
+For modest data volumes where temporary downtime is tolerable, Datastax recommends two open-source data migration tools to help with brief outages:
+
+* Cassandra Data Migrator: The best choice to migrate large data quantities, and where detailed logging, data verifications, table column renaming (if needed), and reconciliation options are provided.
+* DSBulk Migrator: This leverages DataStax Bulk Loader (DSBulk) to perform the data migration, and provides new commands specific to migrations. DSBulk Migrator is ideal for simple migration of smaller data quantities, and where data validation (other than post-migration row counts) is not necessary.
+
+image::{imagesprefix}zdm-migration-phase2.png[Phase 2 diagram shows using tools to migrate data from Origin to Target.]
```

Comment:
I'm not sure why this image is here. It's below the bullet for DSBulk Migrator, but it's not clear that the image is related to that tool.

Reply:
Faizal Rub suggested removing this image.